### PR TITLE
boost: Don't use pre-compiled headers for Fujitsu compiler

### PR DESCRIPTION
**Pull request by tkameyama**

### CODE REVIEW

File: var/spack/repos/builtin/packages/boost/package.py [hunk 1]
```diff
@@ -410,6 +410,8 @@ def determine_b2_options(self, spec, options):
             # change into boost compilation
             if spec.variants['cxxstd'].value == '11':
                 cxxflags.append('-std=c++11')
+        elif spec.satisfies('%fj'):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/boost/package.py]: Would it be better to move this above so we have:
```python
if spec.satisfies('%clang') or spec.satisfies('%fj'):
```

**Review by adamjstewart — COMMENTED** [review]
@hainest can you review this?

File: var/spack/repos/builtin/packages/boost/package.py (hunk 1 above)
Comment (tkameyama) [reply var/spack/repos/builtin/packages/boost/package.py]: fixed.

**Comment by hainest** [comment]
@tkameyama 

Could you change the title to `boost: Don't use pre-compiled headers for Fujitsu compiler`?

Do we know if this is a problem for all versions of the Fujitsu compiler?

**Comment by tkameyama** [comment]
> 
> 
> @tkameyama 
> Do we know if this is a problem for all versions of the Fujitsu compiler?

I'm check only current version of Fujitsu compiler.

**Comment by hainest** [comment]
@tkameyama Ok. We can merge this in and see if other Fujitsu users complain about the PCHs going missing.

@adamjstewart I'm ready to merge this, if you are.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**